lib/std/os/windows.zig

```diff
@@ -1601,6 +1713,67 @@ inline fn MAKELANGID(p: c_ushort, s: c_ushort) LANGID {
     return (s << 10) | p;
 }
 
+pub const WindowsVersion = std.Target.Os.WindowsVersion;
+
+/// Returns the highest known WindowsVersion deduced from reported runtime information.
+/// Discards information about in-between versions we don't differentiate.
+pub fn detectRuntimeVersion() WindowsVersion {
```

Comment:
Is this call really necessary? We already call `std.os.windows.ntdll.RtlGetVersion` in `std.zig.system` and in this line here in particular: [std/zig/system.zig#L229](https://github.com/ziglang/zig/blob/master/lib/std/zig/system.zig#L229). Also, for future reference, any bit of code that works out the target/crosstarget version should go in `system.zig`.